terraform/module-prometheus/outputs.tf

```diff
@@ -13,35 +13,36 @@
 #}
 
 output "prometheus_eip" {
-  value = "${element(aws_eip.prometheus.*.public_ip, 0)}"
+  value = element(aws_eip.prometheus.*.public_ip, 0)
```

Comment:
with terrafrom 0.12 we can now remove element(0) and use `aws_eip.prometheus[0].public_ip`